gradle.properties

```diff
@@ -25,7 +25,7 @@ mavenGroupId         = com.microsoft.graph
 mavenArtifactId      = microsoft-graph-core
```

Comment:
we probably need to change the group id too

Reply:
v1, beta, core, all should have the same group id, I think the maveArtifactId is what should be changing, and I think that is exactly how it is setup at the moment.